upup/models/config/components/kubelet/_networking_cni/_k8s_1_5/kubelet.cni.1_5.options

```diff
@@ -0,0 +1,7 @@
+Kubelet:
+  # AWS MTU is 9001
+  NetworkPluginMTU: 9001
```

Comment:
Hi @chrislovecnm , I got a question about this MTU. According to Weave document, the MTU should be 8950 as there is an overhead of 50 bytes for Vxlan, will this be handled automatically? 

Another question is I can see `kubenet` is using the MTU passed to the plugin (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/kubenet/kubenet_linux.go#L128), however, for cni it doesn't seem it's using the MTU value (https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni.go#L144). Is this been handled somewhere else?